.github/workflows/aws.yml

```
@@ -0,0 +1,17 @@
+$ECR_REPOSITORY:$IMAGE_TAG" >> 
+
+    - name: Fill in the new image ID in the Amazon ECS task definition
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)